src/binder/statement/create_table.rs

```diff
@@ -124,10 +140,14 @@ mod tests {
         let catalog = Arc::new(RootCatalog::new());
         let mut binder = Binder::new(catalog.clone());
         let sql = "
-            create table t1 (v1 int not null, v2 int); 
+            create table t1 (v1 int not null, v2 int);
```

Comment:
We can have planner test in the future 🤣 I guess writing this test case must be painful.

FYI, https://github.com/singularity-data/risingwave/blob/main/src/frontend/test_runner/tests/testdata/tpch.yaml